yaksh/views.py

```diff
@@ -910,14 +923,15 @@ def design_questionpaper(request, quiz_id, questionpaper_id=None):
 
         if 'remove-fixed' in request.POST:
             question_ids = request.POST.getlist('added-questions', None)
-            que_order = question_paper.fixed_question_order.split(",")
-            for qid in question_ids:
-                que_order.remove(qid)
-            if que_order:
-                question_paper.fixed_question_order = ",".join(que_order)
-            else:
-                question_paper.fixed_question_order = ""
-            question_paper.save()
+            if question_paper.fixed_question_order:
```

Comment:
It is required for question papers created without fixed question order. It throws error if tried to remove questions from question paper which does not have fixed question order.